modules/core/cmake/Modules/ConfigureCuGraph.cmake

```diff
@@ -33,66 +39,28 @@ function(find_and_configure_cugraph VERSION)
         set(BUILD_TESTS OFF)
         set(BUILD_BENCHMARKS OFF)
 
+        set(NCCL_LIBRARY "/usr/local/cuda/lib64/libnccl.so")
+        set(NCCL_INCLUDE_DIR "/usr/local/cuda/include")
+
         CPMFindPackage(NAME     cugraph
             VERSION             ${CUGRAPH_VERSION}
-            GIT_REPOSITORY      https://github.com/rapidsai/cugraph.git
-            GIT_TAG             branch-${CUGRAPH_VERSION}
+            # GIT_REPOSITORY      https://github.com/rapidsai/cugraph.git
+            # GIT_TAG             branch-${CUGRAPH_VERSION}
+            GIT_REPOSITORY      https://github.com/robertmaynard/cugraph.git
+            GIT_TAG             use_rapids_cmake
             GIT_SHALLOW         TRUE
             UPDATE_DISCONNECTED FALSE
-            # SOURCE_SUBDIR  cpp
-            DOWNLOAD_ONLY
+            SOURCE_SUBDIR       cpp
+            OPTIONS             "BUILD_TESTS OFF"
+                                "BUILD_BENCHMARKS OFF"
         )
 
-        # set(CUGRAPH_INCLUDE_DIR_REAL "${cugraph_SOURCE_DIR}/cpp/include")
-        # set(CUGRAPH_INCLUDE_DIR "${cugraph_SOURCE_DIR}/cpp/fake_include")
-        # list(APPEND CUGRAPH_INCLUDE_DIRS ${RAFT_INCLUDE_DIR})
-        # list(APPEND CUGRAPH_INCLUDE_DIRS ${CUGRAPH_INCLUDE_DIR})
-        # list(APPEND CUGRAPH_INCLUDE_DIRS ${CUGRAPH_INCLUDE_DIR_REAL})
-        # list(APPEND CUGRAPH_INCLUDE_DIRS ${cugraph_SOURCE_DIR}/cpp/src)
-        # set(CUGRAPH_INCLUDE_DIRS ${CUGRAPH_INCLUDE_DIRS} PARENT_SCOPE)
-        # set(cugraph_SOURCE_DIR "${cugraph_SOURCE_DIR}" PARENT_SCOPE)
-
         execute_process(COMMAND mkdir -p "${cugraph_SOURCE_DIR}/cpp/fake_include")
         execute_process(COMMAND ln -s -f "${cugraph_SOURCE_DIR}/cpp/include" "${cugraph_SOURCE_DIR}/cpp/fake_include/cugraph")
```

Comment:
Remove this craziness once https://github.com/rapidsai/cugraph/pull/1582 is merged.